store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -311,15 +326,15 @@ public CarbonWriterBuilder withTableProperties(Map<String, String> options) {
     }
 
     for (Map.Entry<String, String> entry : options.entrySet()) {
-      if (entry.getKey().equalsIgnoreCase("equalsIgnoreCase")) {
+      if (entry.getKey().equalsIgnoreCase("blocksize")) {
         this.withBlockSize(Integer.parseInt(entry.getValue()));
       } else if (entry.getKey().equalsIgnoreCase("blockletsize")) {
         this.withBlockletSize(Integer.parseInt(entry.getValue()));
       } else if (entry.getKey().equalsIgnoreCase("localDictionaryThreshold")) {
         this.localDictionaryThreshold(Integer.parseInt(entry.getValue()));
       } else if (entry.getKey().equalsIgnoreCase("enableLocalDictionary")) {
         this.enableLocalDictionary((entry.getValue().equalsIgnoreCase("true")));
-      } else {
+      } else if (entry.getKey().equalsIgnoreCase("sortcolumns")) {
```

Comment:
Please make sure all property names are the same as create table properties

Reply:
yes. changed it to same as table properties syntax